### PR TITLE
Latest projects

### PR DESCRIPTION
**Pull request by Vidmasas**
I have managed to transfer my repositories to the work account (wasn't able to push to my personal one after our changes) so no more log in issues 😆 

Finished the last section `Latest Projects`

image

image

After we push this PR, I will try to polish things all around and add footer as a finishing touch 😸 Then you will be able to check all the things to give the insights and things to fix. Thank you!

### CODE REVIEW

**Comment by alynioke** [comment]
`.DS_Store ` and `img/.DS_Store` should be deleted and ignored

File: styles/latest-projects.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.latest-projects-table-header {
+  background-color: rgba(246, 249, 252, 0.03);
+  height: 50px;
+}
+
+.latest-projects-list {
+  height: 49px;
+  border-bottom: solid #212938 1px;
```
Comment (alynioke) [inline styles/latest-projects.css:26]: shorthand has different format https://www.w3schools.com/css/css_border_shorthand.asp

File: styles/latest-projects.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.latest-projects-name {
+  padding-left: 16px;
+}
+
+.lastest-projects-list-name {
```
Comment (alynioke) [inline styles/latest-projects.css:33]: typo. `lastest` -> `latest`

File: styles/latest-projects.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  padding: 10px;
+}
+
+.latest-project-list-icon {
```
Comment (alynioke) [inline styles/latest-projects.css:37]: here it's `project`  and in other classes it's `projects` - if you didn't have any semantical meaning behind that, better to unify and use one ending in all places.

File: index.html [hunk 4]
```diff
@@ -362,10 +362,90 @@
...
                     <!--latest-projects.css-->
                     <div class="latest-projects tab-background-color content-margin tab-structure">
-                        <div class="line1 flex-and-space-between">
+                        <div class="flex-and-space-between">
                             <div class="dashboard-name color-light">Latest Projects</div>
                             <div class="projection-dot-menu"><i class="fas fa-ellipsis-h fa-lg color-dark"></i></div>
                         </div>
+                        <div class="latest-projects-content-padding">
+                            <table class="lastest-projects-content">
+                                <thead style="text-align: left;">
```
Comment (alynioke) [inline index.html:371]: avoid "style" use classes ;)

File: index.html [hunk 5]
```diff
@@ -362,10 +362,90 @@
...
+                                    <tr class="latest-projects-table-header color-light">
+                                        <th class="latest-projects-name">Name</th>
+                                        <th class="latest-projects-project">Project</th>
+                                        <th class="latest-projects-start-date">Start Date</th>
+                                        <th class="latest-projects-end-date">End Date</th>
+                                        <th class="latest-projects-status">Status</th>
```
Comment (alynioke) [inline index.html:377]: maybe remove classnames you're not using in css? as latest-projects-status (or maybe others - didn't check if they exist)

**Comment by Vidmasas** [comment]
> `.DS_Store ` and `img/.DS_Store` should be deleted and ignored

How these appear from time to time? 🤔

File: styles/latest-projects.css (hunk 1 above)
Comment (Vidmasas) [reply styles/latest-projects.css:26]: It seems to be working fine. Should I change the arrangement only? To `border-bottom: 1px solid #212938;`

File: styles/latest-projects.css (hunk 2 above)
Comment (Vidmasas) [reply styles/latest-projects.css:33]: Installed spellchecker, should be fine from now on 🤣

File: styles/latest-projects.css (hunk 3 above)
Comment (Vidmasas) [reply styles/latest-projects.css:37]: You have a very good eye, noticing this mistake 😄

File: index.html (hunk 4 above)
Comment (Vidmasas) [reply index.html:371]: Is it ok to have `thead` styled in css or should I name a class?

File: index.html (hunk 5 above)
Comment (Vidmasas) [reply index.html:377]: Is it good practice if I creating classes just for the structure of the HTML and later removing them? Or should I try not to name the divs into classes when creating the structure and name only if styles are required?

**Comment by alynioke** [comment]
i don't know what it is 😬 but it's generate by your MacOS. 
you can now always check OBerlo Merchant, how it's ignored there ;]

File: styles/latest-projects.css (hunk 1 above)
Comment (alynioke) [reply styles/latest-projects.css:26]: i also checked that it's actually working, but yes, change arrangement to support same style everywhere.

File: styles/latest-projects.css (hunk 2 above)
Comment (alynioke) [reply styles/latest-projects.css:33]: yaay!

File: styles/latest-projects.css (hunk 3 above)
Comment (alynioke) [reply styles/latest-projects.css:37]: thank you! I'm glad you appreciate it 😄

File: index.html (hunk 4 above)
Comment (alynioke) [reply index.html:371]: i am supporter of always using classes :) so don't use `thead`, use class. 

But in Oberlo Merchant you sometimes gonna see tag names and styles applied to `span`  or `a`  in css... i dislike it and discourage it. Why it's used - people make assumption that structure won't change so it's safe. I make worst case  assumption - structure gonna change or tag, + it's better to have same style selectors everywhere.

File: index.html (hunk 5 above)
Comment (alynioke) [reply index.html:377]: ye, leave it only if required. It's not usual practice to create class names "for the future". 
I know it looks cool when all is aligned nicely :D but it's extra noise and can be confusing (e.g. maybe bring throughs "why no class in css? is there bug?")

File: styles/latest-projects.css (hunk 1 above)
Comment (Vidmasas) [reply styles/latest-projects.css]: ok

File: index.html (hunk 4 above)
Comment (Vidmasas) [reply index.html]: True

File: index.html (hunk 5 above)
Comment (Vidmasas) [reply index.html]: Agree on both 😸

**Comment by alynioke** [comment]
1) i see you address thead as tag name, but not like class 
2) ignore DS files in all directories. Example in OM 

the rest of fixes look good

File: styles/latest-projects.css [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+.latest-projects-footer {
+  font-size: 12px;
+  padding: 15px 24px 0;
+}
+
+.latest-projects-entries-previous {
+  padding-right: 16px;
+}
+
+.latest-project-thead {
```
Comment (Vidmasas) [inline styles/latest-projects.css:54]: I just changed it, It should be ok

**Pull request merged**